src/services/juno/custom/helpers/admin-resource.ts

```diff
@@ -0,0 +1,81 @@
+import { AdminResources } from "services/types";
+import { CW3Config } from "types/contracts";
+import { queryContract } from "services/juno/queryContract";
+import { isJunoAddress } from "schemas/tests";
+import { contracts } from "constants/contracts";
+import { adminRoutes, appRoutes } from "constants/routes";
+import { defaultProposalTags } from "../../tags";
+import { customApi } from "../custom";
+
+export const AP_ID = 0;
+export const REVIEWER_ID = 0.5;
+
+export function getCWs(id: number) {
```

Comment:
Suggestion: SInce we only have AP_ID and Review_ID we could create contract object to optimise getCWs
eg:

const CONTRACTS = {
  [AP_ID]: {
    cw3: contracts.cw3ApTeam,
    cw4: contracts.cw4GrpApTeam,
    type: "ap",
  },
  [REVIEWER_ID]: {
    cw3: contracts.cw3ReviewTeam,
    cw4: contracts.cw4GrpReviewTeam,
    type: "review",
  },
};

export function getCWs(id: number) {
  const { cw3, cw4, type } = CONTRACTS[id] || {};
  return { cw3, cw4, type };
}